db/migrate/20190514124322_create_firmware_binaries.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFirmwareBinaries < ActiveRecord::Migration[5.0]
+  def change
+    create_table :firmware_binaries do |t|
+      t.string   :name
+      t.string   :ems_ref
+      t.text     :description
+      t.string   :version
+      t.string   :type
+      t.bigint   :firmware_registry_id
```

Comment:
You can use `t.references :firmware_registry` instead of calling out the exact id column.  You might want to also add `:index => true`